tests/cache/tests.py

```diff
@@ -1144,6 +1144,26 @@ def test_incr_decr_timeout(self):
         cache.decr(key)
         self.assertEqual(expire, cache._expire_info[_key])
 
+    def test_lru_eviction_policy(self):
+        """#28977 -- Ensures items are evicted in LRU-order."""
+        for key in range(300):  # Fill cache to "max entries".
+            cache.set(key, key, timeout=None)
+
+        for key in range(200):  # Move first 200 keys to "front".
+            self.assertEqual(cache.get(key), key)
+
+        cache.set(301, 301, timeout=None)  # Trigger culling.
+
+        # Validate LRU eviction.
+
+        for key in range(200):
+            self.assertEqual(cache.get(key), key)
+
+        for key in range(201, 300):
+            self.assertEqual(cache.get(key), None)
+
+        self.assertEqual(cache.get(301), 301)
```

Comment:
can the test use fewer keys to go faster and be easier to debug?

Reply:
Sure. Decreased from 300 to 30